doc/whats_new.rst

```diff
@@ -69,7 +69,11 @@ Bug fixes
      `#6497 <https://github.com/scikit-learn/scikit-learn/pull/6497>`_
      by `Sebastian Säger`_
 
-
+   - Fixes issue in :class:`_BaseFilter` where score functions were 
```

Comment:
The link will not work because it doesn't include the module, also the class is private, so there's no documentation to link to. Maybe do a ``:ref:`univariate_feature_selection``` instead?